src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/MultiSelectList.cs

```diff
@@ -10,6 +10,11 @@
 
 namespace Microsoft.AspNet.Mvc.Rendering
 {
+    /// <summary>
+    /// Represents a list that lets users select one item.
```

Comment:
Oops, changed single/multiple - I got them backwards.

This isn't the list that users actually select, though. This is just stuff that is later bound.